src/it/scala/com/bwsw/cloudstack/entities/requests/vm/VmCreateRequestIntegrationTestSuite.scala

```diff
@@ -0,0 +1,88 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package com.bwsw.cloudstack.entities.requests.vm
+
+import java.util.UUID
+
+import br.com.autonomiccs.apacheCloudStack.client.ApacheCloudStackRequest
+import com.bwsw.cloudstack.entities.TestEntities
+import com.bwsw.cloudstack.entities.requests.account.AccountFindRequest
+import com.bwsw.cloudstack.entities.util.responses.vm.{VmCreateResponse, VmTest, VmTestFindResponse}
+import com.bwsw.cloudstack.entities.util.responses.account.AccountTestFindResponse
+import org.scalatest.FlatSpec
+
+class VmCreateRequestIntegrationTestSuite extends FlatSpec with TestEntities {
+  val serviceOfferingId = retievedServiceOfferingId
+  val templateId = retrievedTemplateId
+  val zoneId = retrievedZoneId
+
+  it should "create vm use request which contains only required parameters" in {
+    val vmCreateRequest = new VmCreateRequest(VmCreateRequest.Settings(serviceOfferingId, templateId, zoneId))
+
+    checkVmCreation(vmCreateRequest.request)
+  }
+
+  it should "create vm use request which contains all available parameters" in {
```

Comment:
contains the required and optional parameters

Reply:
ok